wagtail_ab_testing/static/wagtail_ab_testing/js/tracker.js

```diff
@@ -1,6 +1,6 @@
 (function() {
     // Check if Do Not Track is enabled
-    if (window.doNotTrack || navigator.doNotTrack || navigator.msDoNotTrack || 'msTrackingProtectionEnabled' in window.external) {
+    if (window.doNotTrack || navigator.doNotTrack || navigator.msDoNotTrack || (window.external || {}).msTrackingProtectionEnabled) {
         if (window.doNotTrack == '1' || navigator.doNotTrack == 'yes' || navigator.doNotTrack == '1' || navigator.msDoNotTrack == '1' || 'msTrackingProtectionEnabled' in window.external && window.external.msTrackingProtectionEnabled()) {
```

Comment:
I think we are good dropping the entire `window.external. msTrackingProtectionEnabled` check.

This appears to be only supported in Internet Explorer 9 if this docs page I found is accurate: https://learn.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/platform-apis/gg593071(v=vs.85)